### ISSUE TITLE
Section 6.5.3, page 204

### ISSUE THREAD

**Issue by jordiclive** [mml-book/mml-book.github.io#398]
**Describe the mistake**
"Let $\bm{y}\in \mathbb{R} ^{m}$ be a Gaussian random variable with mean \bm{Ax}"

If this is the case then shouldn't 6.9 and 6.91, exchange $\bm{y}$ for $\bm{\mu _{y}}$

**Location**

1. Draft (2019-08-20
2. 6.5.3
3. 204
4. 6.9 and 6.91

**Comment by aliunlu97** [comment]
In equation 6.89, it states that p(y)=\mathcal{N}(y|Ax,\Sigma) meaning that y is distributed according to the Gaussian Distribution with a mean of Ax. Hence, \mu_{y}=Ax
Since y = Ax, the pseudo inverse has been multiplied with y. I hope this helps.

Sincerely,

**Comment by mpd37** [comment]
The idea here is that we want to isolate x from the relation y = Ax. Under some mild assumptions we can do this by 
y = Ax  => A^Ty = A^T A x => (A^T A)^{-1} A^T y = x

I'm not sure whether this helps?

**Comment by arya-map22** [comment]
> The idea here is that we want to isolate x from the relation y = Ax. Under some mild assumptions we can do this by y = Ax => A^Ty = A^T A x => (A^T A)^{-1} A^T y = x
> 
> I'm not sure whether this helps?

Yes, but in the 6.91 it said that the E(X) = (A^T A)^{-1} A^T y, with y a state of random variable Y. And from definition of Y, E(Y)=Ax (page 204) not y = Ax (page 203), so i got confused here. How come the mean of X become like that?
